backend/api/v2beta1/recurring_run.proto

```diff
@@ -70,6 +71,82 @@ service RecurringRunService {
   }
 }
 
+message RecurringRun {
```

Comment:
Maybe do not move the location of this message definition so we keep the change history.

Reply:
This is a good point. Although there is a very short history given that this is a new file. The motivation for the change is to have the same order in the proto files: 1) services, 2) main messages, 3) request and response messages. If you or @chensun have a strong preference, I can rearrange it. Please note that I also replaced PipelineId with PipelineVersionId and introduced a few less important changes.